packages/teams-js/src/private/interfaces.ts

```diff
@@ -271,3 +271,142 @@ export interface UserJoinedTeamsInformation {
    */
   userJoinedTeams: TeamInformation[];
 }
+
+/**
+ * @hidden
+ * Eligibility Information for the app user.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * @beta
+ */
+export interface AppEligibilityInformation {
+  /**
+   * Describes the user’s chat experience based on their eligible licenses & their tenant’s eligible licenses.
+   * A user will be in at most one cohort.
+   */
+  cohort: Cohort;
+  /**
+   * Describes additional traits of the user that contribute to FRE experience, etc.
+   */
+  persona: Persona;
+  /**
+   * Describes the user’s age group, which can have implications on which product they are able to use.
+   */
+  ageGroup: LegalAgeGroupClassification;
+  /**
+   * Implementation may change to be based on tenant-home region rather than IP.
+   */
+  isCopilotEnabledRegion: boolean;
+  /**
+   * Indicates that the user is eligible for some AAD authenticated Copilot experience.
+   */
+  isCopilotEligible: boolean;
+  /**
+   * Indicates if the tenant admin has opted the user out of Copilot.
+   */
+  isOptedOutByAdmin: boolean;
+  /**
+   * For EDU tenants only. Indicates if the tenant is higher ed or K12.
+   */
+  eduType: EduType;
```

Comment:
>EduType [](http://example.com/codeflow?start=11&length=7)

Rather than having a `None` member of `EduType`, it would be clearer if this was declared as:
```
eduType: EduType | null;
```
...and setting the property to `null` when there is no EDU tenant. If you have an `EduType` object instance, the implication is that you can only get one in an applicable EDU tenant.